lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +45,33 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def enough_approvals?
```

Comment:
We'll need test coverage on this.  Also I think we can consolidate them into a single `approved?` method.